Cilsil/Program.cs

```diff
@@ -127,9 +129,12 @@ private static void Main(string[] args)
         /// </summary>
         /// <param name="paths">The paths.</param>
         /// <param name="printprocs">The printprocs.</param>
+        /// <param name="outelapsetime">The elapse time output path (used for visualizing the elapse 
+        /// time per method).</param>
         /// <returns></returns>
         public static (Cfg, TypeEnvironment) ExecuteTranslation(string[] paths,
-                                                                string printprocs = null)
+                                                                string printprocs = null,
+                                                                string outelapsetime = null)
         {
             var watch = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
we don't want to start this resource at all when the flag is off

Reply:
Done